gcloud/monitoring/timeseries.py

```diff
@@ -127,6 +127,7 @@ class Point(collections.namedtuple('Point', 'end_time start_time value')):
     :type start_time: string or None
     :param start_time: An optional start time in RFC3339 UTC "Zulu" format.
 
+    :type value: int
```

Comment:
In `gcloud-python`, we have type annotations for all values (except when they fall through the cracks like this). A note like "any" or being literal and saying `object` or just giving the list of possible types @rimey just listed should all be fine.